Some clarifications and cleanup for custom domains and service bindings (#5927)
cloudflare/cloudflare-docs: content/workers/platform/bindings/about-service-bindings.md

![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](/workers/platform/bindings/media/service-bindings-config.png)

### Wrangler

To configure a Service binding in your `wrangler.toml`, use the following syntax:

```toml
services = [

>   { binding = "<binding_name>", service = "<worker_name>", environment = "<environment_name>" }

Reviewer: ```suggestion
  { binding = "<BINDING_NAME>", service = "<WORKER_NAME>", environment = "<ENVIRONMENT_NAME>" }
```

]
```

> The `wrangler.toml` options are:
> * `binding`: variable name for the binding in your Worker code, accessible under the `env` parameter in [Module syntax](/workers/learning/migrating-to-module-workers/), or in the global scope in [Service Worker syntax](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API)
> * `service`: name of the target Worker you'd like to communicate with. This Worker should be on your Account
> * `environment` (optional): name of the environment on the target Worker you'd like to communicate with. This is optional, by default the 'Default' environment will be inferred

Reviewer: ```suggestion
The `wrangler.toml` options are:

* `binding`: Variable name for the binding in your Worker code, accessible under the `env` parameter in [Module syntax](/workers/learning/migrating-to-module-workers/), or in the global scope in [Service Worker syntax](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API).
* `service`: Name of the target Worker you would like to communicate with. This Worker should be on your account.
* `environment` (optional): Name of the environment on the target Worker you would like to communicate with. This is optional, by default the Default environment will be inferred.
```

> ### `wrangler.toml` example
> For the example outlined above, a `wrangler.toml` might look like this:

Reviewer: ```suggestion
### `wrangler.toml` example

For the example outlined above, a `wrangler.toml` might look like this:
```

```toml
services = [
  { binding = "auth", service = "authentication" },
  { binding = "logout", service = "logout", environment = "production" }
]
```

In the example above, the Service bindings for the `authentication` and `logout` Workers are accessible in code via `env.auth` and `env.logout`, respectively (when using Module syntax).

### Local development

Local development is supported for Service bindings. For each Worker, open a terminal and use [`wrangler dev --local`](/workers/wrangler/commands/#dev) in the relevant directory or use the `SCRIPT` option to specify the relevant Worker's entrypoint.

> ### Use Service bindings
> Service bindings are available in your Worker code under the `<binding_name>` specified in `wrangler.toml`, via API, or Dashboard. For example, a Service binding that's named "auth" will be available under the binding name `auth` in your Worker. The API `fetch()` is implemented on each Service binding by default.

Reviewer: ```suggestion
### Use Service bindings

Service bindings are available in your Worker code under the `<BINDING_NAME>` specified in `wrangler.toml`, via API, or the dashboard. For example, a Service binding that is named "auth" will be available under the binding name `auth` in your Worker. The API `fetch()` is implemented on each Service binding by default.
```
